app/controllers/users_controller.rb

```diff
@@ -252,6 +255,15 @@ def init_fields
     @foster_users = User.where(locked: false).order("name")
   end
 
+  def active_user
+    redirect_to(root_path) unless current_user.active?
+  end
+
+  def show_other_users
```

Comment:
With the first condition being `current_user?(@user)` it comes across as a bit confusing to me that this is called `show_other_users` - the same check is applied to view one's own profile. Perhaps something like `allowed_to_see_user` or similar? Naming is hard...